locales/en-US/app.ftl

```diff
@@ -102,6 +103,84 @@ ListOfPublishedProfiles--uploaded-profile-information-list =
+MenuButtons--metaInfo--resymbolicate-profile = Re-symbolicate profile
+MenuButtons--metaInfo--symbolicate-profile = Symbolicate profile
+MenuButtons--metaInfo--attempting-resymbolicate = Attempting to re-symbolicate profile
+MenuButtons--metaInfo--currently-symbolicating = Currently symbolicating profile
+MenuButtons--metaInfo--recording-started = Recording started:
+MenuButtons--metaInfo--interval = Interval:
+MenuButtons--metaInfo--profile-version = Profile Version:
+MenuButtons--metaInfo--buffer-capacity = Buffer Capacity:
+MenuButtons--metaInfo--buffer-duration = Buffer Duration:
+
+# Buffer Duration in Seconds in Meta Info Panel
+# Variable:
+#   $configurationDuration (Number) - Buffer Duration in Seconds
+MenuButtons--metaInfo--buffer-duration-seconds = { $configurationDuration } seconds
+MenuButtons--metaInfo--buffer-duration-unlimited = Unlimited
+MenuButtons--metaInfo--application = Application
+MenuButtons--metaInfo--name-and-version = Name and version:
+MenuButtons--metaInfo--update-channel = Update Channel:
+MenuButtons--metaInfo--build-id = Build ID:
+MenuButtons--metaInfo--build-type = Build Type:
+MenuButtons--metaInfo--platform = Platform
+MenuButtons--metaInfo--visual-metrics = Visual Metrics
+MenuButtons--metaInfo--speed-index = Speed Index:
+MenuButtons--metaInfo--perceptual-speed-index = Perceptual Speed Index:
+MenuButtons--metaInfo--contentful-speed-Index = Contentful Speed Index:
+
+MenuButtons--metaOverheadStatistics-subtitle = Profiler Overhead
+MenuButtons--metaOverheadStatistics-mean = Mean
+MenuButtons--metaOverheadStatistics-max = Max
+MenuButtons--metaOverheadStatistics-min = Min
+MenuButtons--metaOverheadStatistics-overhead-duration = Overhead Durations:
+MenuButtons--metaOverheadStatistics-overhead-percentage = Overhead Percentage:
+MenuButtons--metaOverheadStatistics-profiled-duration = Profiled Duration:
+
+MenuButtons--permalink--button =
+    .label = Permalink
+
+MenuButtons--publish--reupload-performance-profile = Re-upload Performance Profile
+MenuButtons--publish--share-performance-profile = Share Performance Profile
+MenuButtons--publish--info-description = Upload your profile and make it accessible to anyone with the link.
+MenuButtons--publish--info-description-default = By default, your personal data is removed.
+MenuButtons--publish--info-description-firefox-nightly = This profile is from { -firefox-nightly-brand-name }, so by default all information is included.
+MenuButtons--publish--include-additional-data = Include additional data that may be identifiable
+MenuButtons--publish--button-upload = Upload
+MenuButtons--publish--upload-title = Uploading profile…
+MenuButtons--publish--cancel-upload = Cancel Upload
+MenuButtons--publish--message-something-went-wrong = Uh oh, something went wrong when uploading the profile.
+MenuButtons--publish--message-try-again = Try again
+MenuButtons--publish--download = Download
+MenuButtons--publish--compressing = Compressing…
```

Comment:
@canova I didn't ask about this. So the components inside `MenuButtons` are grouped together here. I was not sure if should add these as other components. Let me know what you think.